src/openMVG/system/cpu_instruction_set.hpp

```diff
@@ -117,9 +119,14 @@ class CpuInstructionSet
 private:
   static bool internal_cpuid(int32_t out[4], int32_t x)
   {
+    #if defined  __aarch64__
+    return false;
+    #endif
     #if defined __GNUC__
-    __cpuid_count(x, 0, out[0], out[1], out[2], out[3]);
-    return true;
+      #if not defined __aarch64__
```

Comment:
This one should not be needed, since you already return false for aarch64, right?

Reply:
yeah, but the compiler thinks this is an error:

![image](https://github.com/openMVG/openMVG/assets/61532079/5c13c173-a9a8-4fb2-bbdc-792f09f6ea9e)

Reply:
So we could have a unique if defined aarch64 return false here and remove the one on top, right?

Reply:
@jujimeizuo could you update the code to have only one entry with `__aarch64__` so it would ease later maintenance?

Reply:
yep